requirements/common_constraints.txt

```diff
@@ -24,7 +24,3 @@
 elasticsearch<7.14.0
 
 # django-simple-history>3.0.0 adds indexing and causes a lot of migrations to be affected
-
-# tox>4.0.0 isn't yet compatible with many tox plugins, causing CI failures in almost all repos.
-# Details can be found in this discussion: https://github.com/tox-dev/tox/discussions/1810
-tox<4.0.0
```

Comment:
Why are we removing this constraint?

Reply:
@mariajgrimaldi this occurred automatically after running `make compile-requirements`